File: js/programmers/level.0/prgm_181921_1.js
function solution(l, r) {
  let result = [];
  for (i = l; i <= r; i++) {
    let num = String(i);
    if ([...num].every((x) => x === "5" || x === "0")) result.push(i);
  }
  return result.length ? result : [-1];
}

File: python/programmers/level.0/prgm_181921_1.py
def solution(l, r):
    answer = []

    for i in range(l, r+1):
        num = str(i)
        if all(x == "5" or x == "0" for x in [*num]):
            answer.append(i)
        
    return answer if len(answer) > 0 else [-1]